test/Core.Test/Vault/AutoFixture/CollectionFixture.cs

```diff
@@ -0,0 +1,52 @@
+﻿using AutoFixture;
+using Bit.Core.Context;
+using Bit.Core.Entities;
+using Bit.Core.Models.Data;
+using Bit.Core.Test.AutoFixture;
+using Bit.Test.Common.AutoFixture.Attributes;
+
+namespace Bit.Core.Test.Vault.AutoFixture;
+
+public class CollectionCustomization : ICustomization
+{
+    private const int _collectionIdSeed = 1;
+    private const int _userIdSeed = 2;
+    private const int _groupIdSeed = 3;
+
+    public void Customize(IFixture fixture)
+    {
+        var orgIds = new[] { Guid.NewGuid(), Guid.NewGuid() };
```

Comment:
When playing with this, I found that autofixture always generates 3 organizations using `BitAutoData`, which means it'll wrap around to the start of this array and the you'll end up with the first and last org sharing the same Guid. I solved this by adding another `Guid.NewGuid()` element here, but maybe there's a better way.

Reply:
With the change to only support single orgs this became no longer necessary and I've switched it back to a single `orgId`. (This was another complexity that was pulling me away from trying to support multiple orgs in the same request)